# This must be the name of the robot namespace
trough_bot:
  # Publish all joint states (for feedback?)
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position controllers
  arm_trough_back_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_trough_back_joint
    pid: {p: 1.0, i: 0.00, d: 00.0}